.github/workflows/release-notes.yml

```diff
@@ -6,34 +6,18 @@ on:
   push:
     tags:
       - v*
-  workflow_run:
-    workflows: ['Tag on Merge from Release Branch']
-    types:
-      - completed
+  workflow_dispatch:
 
 jobs:
   release_notes:
     runs-on: ubuntu-24.04
 
     steps:
-      - name: Create GitHub App Token
-        uses: actions/create-github-app-token@v1
-        id: app-token
-        with:
-          app-id: ${{ vars.PR_AUTOMATION_BOT_PUBLIC_APP_ID }}
-          private-key: ${{ secrets.PR_AUTOMATION_BOT_PUBLIC_PRIVATE_KEY }}
-
       - name: Checkout repository
         uses: actions/checkout@v4
 
       - name: Publish Release Notes
```

Comment:
Note the safety check: We use `ref_name` but require `ref_type` to be a tag.  See the docs for ref_type: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs